rdr_service/genomic/genomic_job_controller.py

```diff
@@ -246,22 +251,24 @@ def ingest_specific_manifest(self, filename):
         except RuntimeError:
             self.job_result = GenomicSubProcessResult.ERROR
 
-    def ingest_member_ids_from_aw1_raw_table(self, member_ids):
+    def ingest_member_ids_from_awn_raw_table(self, n, member_ids):
         """
         Pulls AW1 data from genomic_aw1_raw and loads to genomic_set_member
+        :param n: 1 or 2 for AW1, AW2
```

Comment:
Is there ever going to be a need for more than just these two raw tables?

Reply:
There may be more raw tables in the future, but the AW1 and AW2 would still be the primary tables that we would source data from using these tools.

Reply:
(Nit) It made me wonder whether another parameter name may be more descriptive/extensible should the use cases for this grow.